overlore/config.py

```diff
@@ -35,8 +60,9 @@ def _get_args(self) -> None:
         parser.add_argument("-a", "--address", help="Host address for ws connection", type=str, default="localhost")
         parser.add_argument("-p", "--port", help="Host port for ws connection", type=int, default=8766)
         parser.add_argument("-w", "--world_db", help="location of the world db", type=str, default="/litefs/world.db")
-
+        parser.add_argument("-l", "--logging_file", help="location of the logging file", type=str)
```

Comment:
Not sure what the behaviour looks like if no path is specified, does it crash? Maybe we should add a default path for this one?

Reply:
if there's no file, it will write to the console. see `setup_logging`